fth/system.fth

```diff
@@ -360,6 +360,10 @@
         2* swap
 ;
 
+: D= ( xd1 xd2 -- flag )
+	rot = -rot = and
```

Comment:
I suppose shorter is faster, so yes.  Hmm, but it would have to be `D- D0=` and we don't have `D0=`. I'm also bit usure what happens on underflow.